source/documentation/introduction_to_the_vm_portal/topics/Connecting_to_virtual_machines.adoc

```diff
@@ -13,3 +13,8 @@
 . Open the file with *remote-viewer*. A console window displays.
 +
 You can now use the virtual machine in the same way you would use a physical desktop.
+
+[NOTE]
+====
+To allow other users to connect to the VM, make sure you restart the virtual machine when you are finished using the console. Alternatively, the administrator can *Disable strict user checking* to eliminate the need for reboot between users. See link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#Virtual_Machine_Console_settings_explained[Virtual Machine Console Settings Explained] in the _Virtual Machine Management Guide_ for more information.
```

Comment:
The same here, state 'shutdown and restart' instead of 'restart'

Reply:
Changes made.